eval-server/nodejs/src/lib/OpenAICompatibleWrapper.js

```
@@ -0,0 +1,700 @@
+ * ```
+ */
+export class OpenAICompatibleWrapper {
+  constructor(evalServer, options = {}) {
+    this.evalServer = evalServer;
+    this.config = {
+      port: options.port || 8081,
+      host: options.host || 'localhost',
+      modelCacheTTL: options.modelCacheTTL || 300000, // 5 minutes in ms
+      ...options
+    };
+    
```

ESLint

[error] Trailing spaces not allowed.

(@stylistic/no-trailing-spaces)